src/Components/Layout/NavBar.tsx

```diff
@@ -9,16 +9,29 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 // import { Avatar, deepOrange } from "@material-ui/core"; (later use)
 
+interface NavBarProps {
+  dashboard: boolean;
+  signOut(): void;
+}
+
+interface NavBarState {
+  Navlink: any[];
```

Comment:
we can easily set a type for the Navlinks

type navlink {
     name: string,
     link: string,
}

and then we can use that

Navlink: navlink[],